tests/cypress/forms_spec.js

```diff
@@ -12,97 +12,110 @@ const getIframeBody = () => {
     .then(cy.wrap);
 };
 
-context("Marketo form", () => {
+context("Marketo forms", () => {
   beforeEach(() => {
-    cy.server();
-    cy.route("POST", "/marketo/submit").as("captureLead");
+    cy.intercept(
+      { method: "POST", url: "/marketo/submit",},
+      { 
+          headers: {
+          "Client Id": Cypress.env("Client Id"),
+          "Client Secret": Cypress.env("Client Secret"),
+          "Authorized User": Cypress.env("Authorized User"),
+          "Token": Cypress.env("Token"),
+        },
+      }
+    ).as("captureLead");
+    cy.setCookie("_cookies_accepted", "all");
   });
 
-  // afterEach(() => {
-  //   cy.wait("@captureLead").should((xhr) => {
-  //     expect(xhr.method).to.equal("POST");
-  //     expect(xhr.status).to.equal(200);
-  //   });
-  // });
-
-  it("/download/server/thank-you", () => {
-    cy.visit("/download/server/thank-you");
-
-    cy.get('input[name="firstName"]').type("Test");
-    cy.get('input[name="lastName"]').type("Test");
-    cy.get('input[name="company"]').type("Test");
-    cy.get('input[name="title"]').type("Test");
-    cy.get('input[name="email"]').type("test@test.com");
-
-    getIframeBody().find(".rc-anchor-content").click();
-
-    cy.wait(10000); // eslint-disable-line
-    cy.get("#mktoForm_3485").submit();
+  afterEach(() => {
+    cy.wait("@captureLead").then(({ request, response }) => {
+      expect(request.method).to.equal("POST");
+      expect(response.statusCode).to.equal(200);
+    });
   });
 
-  it("/core/contact-us", () => {
+  it("should successfully complete contact form and submit to Marketo", () => {
     cy.visit("/core/contact-us");
 
-    cy.get('input[name="firstName"]').type("Test");
-    cy.get('input[name="lastName"]').type("Test");
-    cy.get('input[name="email"]').type("test@test.com");
-    cy.get('input[name="phone"]').type("000000000");
-    cy.get('select[name="Country"]').select("Colombia");
-    cy.get('input[name="company"]').type("Test");
+    cy.findByLabelText(/First name:/).type("Test");
+    cy.findByLabelText(/Last name:/).type("Test");
+    cy.findByLabelText(/Email address:/).type("test@test.com");
+    cy.findByLabelText(/Mobile\/cell phone number:/).type("07777777777");
+    cy.findByLabelText(/Country:/).select("Colombia");
+    cy.findByLabelText(/Company:/).type("Test");
+    // cy.findByLabelText(/Job title:/).type("Test");
     cy.get('input[name="title"]').type("Test");
```

Comment:
Thanks for your review @petermakowski - I've made the tests more user centric. Just one problem I couldn't solve. here for some reason doing `cy.findByLabelText(/Job title:/).type("Test");` wasn't finding the element even though that is the label text. Am I missing something really obvious?

Reply:
Oh interesting. This test actually caught a bug on this page!

Element IDs should be unique. Looks like there are multiple elements with an id "title" that the label refers to. 

![image](https://user-images.githubusercontent.com/7452681/132371119-f278713c-763d-4d58-875e-aa7cf3891c84.png)
![image](https://user-images.githubusercontent.com/7452681/132371148-ef313e71-07f9-423a-a056-9723080d8a33.png)

I'd suggest updating the id of the label to be unique, for example to "job-title" instead of "title".